app/bundles/CampaignBundle/Entity/LeadEventLog.php

```diff
@@ -16,12 +16,12 @@ class LeadEventLog implements ChannelInterface
     private $id;
 
     /**
-     * @var Event|null
+     * @var Event
```

Comment:
Just to double check. There is the code where the event field is not initialized for a new LeadEventLog object. But I can't test it manually, and the tests work fine. 
Getter still returns null:
```
/**
 * @return Event|null
 */
 public function getEvent()

Reply:
@volha-pivavarchyk That is fine, I relaxed `phpstan` settings in this commit 03915a348dadafe9699c8f696bcd18f6e3ed3b5d, so this should be fine